utils/estimated_histogram.hh

```diff
@@ -73,40 +73,58 @@ struct estimated_histogram {
 
     int64_t _count = 0;
```

Comment:
This isn't "Clean estimated histogram".

Reply:
Are you talking about the PR header?
The patch is about getting better estimation

Reply:
Yes. The subject should summarize what the patches are doing.